report.c

```diff
@@ -238,6 +238,8 @@ void report_grbl_settings() {
   printPgmString(PSTR(" (magazine gap enabled, bool)"));
   printPgmString(PSTR("\r\n$42=")); print_uint8_base10(settings.use_load_cell);
   printPgmString(PSTR(" (use load cell, bool)"));
+  printPgmString(PSTR("\r\n$43=1"));
```

Comment:
The way Motion parses this is by index. The index of `end_of_settings` needs to directly follow the valid settings.

If a new settings is added. The index of `end_of_settings` needs to be updated as well.